src/Trixi.jl

```diff
@@ -87,6 +87,9 @@ import SummationByPartsOperators: integrate, semidiscretize,
                            Quad, Hex, Tet, Wedge
 using StartUpDG: RefElemData, MeshData, AbstractElemShape
 
+# For GPU offloading
+using KernelAbstractions
```

Comment:
Usually we try to avoid general `using`s, but instead explicitly list the symbols that we use from a given package, e.g.,
```julia
using KernelAbstractions: @kernel, @index
```

Reply:
I will change that then. Thanks!